automations/python/models/label_group.py

```diff
@@ -1,6 +1,11 @@
+# noqa: D100
+
+
```

Comment:
It seems like this might be unnecessary since it's being ignored at the pre-commit level?
```suggestion
```